.travis/before_install.sh

```diff
@@ -42,7 +42,8 @@ pip install -r dev_requirements.txt
 # check the commit message
 ./.travis/check_commit.sh
 
-
+# check the coverage file update
+./.travis/check_coverage_file.sh
```

Comment:
@dkliban, @daviddavis  since it's RPM specific should we put it in the post_before_install script? Basically the one which is not managed by plugin_template?

Reply:
I first use `pre_before_install.sh` but that one is sourced (`source`) but when use `exit 0` it exit whole `before_install.sh`. But can be refactored if it can be issue.

Reply:
Yea, this will get overwritten the next time we update travis files from plugin_template. I think you can use `return` in place of `exit 0`.